clintrials/dosefinding/efftox.py

```
@@ -752,21 +853,29 @@ def tabulate(self):
         """
         if n is None:
             n = self.num_integral_steps
         cases = list(zip(self._doses, self._toxicities, self._efficacies))
+
+        limit_args = {
+            "k_sd": kwargs.get("k_sd", self.k_sd),
+            "max_iter": kwargs.get("max_iter", self.max_iter),
+            "mass_threshold": kwargs.get("mass_threshold", self.mass_threshold),
+        }
+
         post_params, pds = efftox_get_posterior_params(
```

Ruff (0.15.18)

[warning] Unpacked variable `pds` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)